.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__/
 
 # Distribution / packaging
 *.egg-info/
+odxtools/version.py
```

Comment:
Why?

Reply:
failed experiment: I wanted git to ignore changes to the file even if it was tracked (i.e., so that the file is present even if `pip install -e .` has been forgotten, but changes to it are not shown in `git diff`. that doesn't work that easily with git. anyway, fixed: 18257b6cb1663e1ea326f3554d0701b0d741fa0b